packages/web/components/Patron/Join/PerksGrid.tsx

```diff
@@ -84,272 +94,344 @@ const LeftArrowStyle = {
+        float={{ base: 'none', md: 'left' }}
+        _before={LeftArrowStyle}
       >
-        <PerksHeader title={'Silver League'} count={14} amount={177} />
-
-        <Box p={8} width="100%" color="white">
-          <List textAlign="left">
+        <VStack
+          bg="whiteAlpha.200"
+          spacing={0}
+          borderRadius={8}
+          minH="10rem"
+          mb={{ base: 0, md: '1em' }} // the arrow on the next Box arrow makes the margin at base breakpoint
+        >
+          <PerksHeader
+            title={'Bronze League'}
+            count={getLeagueCount(PlayerRank_Enum.Bronze)}
+            pSeeds={leaguePSeedsByRank[PlayerRank_Enum.Bronze]}
+            amountUsd={
+              pSeedPrice
+                ? leaguePSeedsByRank[PlayerRank_Enum.Bronze] * pSeedPrice
+                : null
+            }
+          />
+          <Box p={8} width="100%" color="white">
             <Flex width="100%" flexDirection="row" flexWrap="wrap">
-              {SilverLeagueList.map((text: string) => (
-                <LeagueCardItem text={text} />
+              {BronzeLeagueList.map((text: string, index) => (
+                <LeagueCardItem key={index} text={text} />
               ))}
             </Flex>
-          </List>
-        </Box>
-      </VStack>
-    </Box>
+          </Box>
+        </VStack>
+      </Box>
 
-    <Box
-      width={{ base: '100%', md: '50%' }}
-      float={{ base: 'none', md: 'left' }}
-      _before={LeftArrowStyle}
-    >
-      <VStack
-        spacing={0}
-        borderRadius={8}
-        minH="8rem"
-        bg="whiteAlpha.200"
-        mb={{ base: 0, md: '1em' }} // the arrow on the next Box arrow makes the margin at base breakpoint
+      <Box
+        width={{ base: '100%', md: '50%' }}
+        float={{ base: 'none', md: 'right' }}
+        _before={RightArrowStyle}
       >
-        <PerksHeader title={'Gold League'} count={7} amount={519} />
+        <VStack
+          spacing={0}
+          borderRadius={8}
+          minH="8rem"
+          bg="whiteAlpha.200"
+          mb={{ base: 0, md: '1em' }} // the arrow on the next Box arrow makes the margin at base breakpoint
+        >
+          <PerksHeader
+            title={'Silver League'}
+            count={getLeagueCount(PlayerRank_Enum.Silver)}
+            pSeeds={leaguePSeedsByRank[PlayerRank_Enum.Silver]}
+            amountUsd={
+              pSeedPrice
+                ? leaguePSeedsByRank[PlayerRank_Enum.Silver] * pSeedPrice
+                : null
+            }
+          />
 
-        <Box p={8} width="100%" color="white">
-          <Flex width="100%" flexDirection="row" flexWrap="wrap">
-            {GoldenLeagueList.map((text: string) => (
-              <LeagueCardItem text={text} />
-            ))}
-          </Flex>
-        </Box>
-      </VStack>
-    </Box>
+          <Box p={8} width="100%" color="white">
+            <List textAlign="left">
+              <Flex width="100%" flexDirection="row" flexWrap="wrap">
```

Comment:
`flex-direction` is `row` by default, right?

```suggestion
              <Flex width="100%" flexWrap="wrap">
```

Reply:
Perhaps, but if you make it explicit, the reader of this code doesn't need to know what the defaults are :)